### PR TITLE
feat(UI-956): functions select in manual run

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
**Single-shot and triggers missing entry-point function names in buildInfo runtimes exports**
Instead of clear text - display a dropdown with all functions coming from the build info (BuildService.Describe function).

## Linear Ticket
https://linear.app/autokitteh/issue/UI-956/manual-run-add-the-ability-to-select-function-out-of-a-dropdown

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/pages/project.tsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const Project = () => {
 	const loadProject = async (projectId: string) => {
 		if (currentProjectId === projectId) return;
 		await initCache(projectId, true);
+		fetchManualRunConfiguration(projectId!);
```
Comment (J1za) [inline src/components/pages/project.tsx:28]:
```suggestion
		fetchManualRunConfiguration(projectId);
```

**Review by J1za — COMMENTED** [review]
On new project after deploy or build, button Manual run still disable 
image

**Comment by RonenMars** [comment]
> On new project after deploy or build, button Manual run still disable

On a new project after deploy manual run (config) is active. After build it shouldn't be active.

**Review by J1za — APPROVED** [review]

File: src/components/pages/project.tsx (hunk 1 above)
Comment (RonenMars) [reply src/components/pages/project.tsx]: done

**Pull request merged**